react/Topic.tsx

```diff
@@ -1,17 +1,17 @@
-import React, { Fragment } from 'react'
+import React, { Fragment, FunctionComponent } from 'react'
 import { Helmet } from 'vtex.render-runtime'
 
 import ErrorPage from './components/ErrorPage'
 import topics from './components/topics'
 
 interface Props {
   params: {
-    id?: string
-    topic: string
+    id?: string;
```

Comment:
Is this semicolon really necessary?

Reply:
They were added by linter hahaha
But it kinda makes sense, because I'm sort of defining a type inside an interface

Reply:
Apparently Prettier did that. I don't know if I should remove them or not hahaha

Reply:
hahaha it happens, but you think it makes sense, then it's ok for me. I just asked because other declarations were not using them